### PR TITLE
[PBA-4626] default controls to hidden initially, and support autohide…

### PR DESCRIPTION
**Pull request by lental**
… = false

### CODE REVIEW

File: sdk/react/ooyalaSkinCore.js [hunk 1]
```diff
@@ -153,11 +153,15 @@ OoyalaSkinCore.prototype.handleVideoTouch = function(event) {
  * Hard reset lastPressedTime, either due to button press or otherwise
  */
 OoyalaSkinCore.prototype.handleControlsTouch = function() {
-  Log.verbose("handleControlsTouch - Time reset");
-  this.skin.setState({lastPressedTime: new Date().getTime()});
+  if (this.skin.props.controlBar.autoHide === true) {  
+    Log.verbose("handleVideoTouch - Time set");
```
Comment (brol1dev) [inline sdk/react/ooyalaSkinCore.js:157]: Do we want these Logs?

**Review by brol1dev — APPROVED** [review]
LGTM, I just added one comment in a file.

File: sdk/react/ooyalaSkinCore.js (hunk 1 above)
Comment (lental) [reply sdk/react/ooyalaSkinCore.js:157]: logs are fine - after compiling the jsbundle, logs are removed.  futhermore, events that happen on-click are fine to log out to console, as it won't overflow the console output and gives good insight what happens

Comment (brol1dev) [reply sdk/react/ooyalaSkinCore.js:157]: ok, that's all then :)

**Pull request merged**